tests/unittests/tiglFuselage.cpp

```diff
@@ -188,3 +191,34 @@ TEST(TiglSimpleFuselage, getSurfaceArea_HalfModel)
     tiglCloseCPACSConfiguration(tiglHandle);
     tixiCloseDocument(tixiHandle);
 }
+
+TEST(TiglSimpleFuselage, GetPointTangent_checkArgs)
+{
+    const char* logfileName = "LogFile_GetPointTangent_checkArgs";

```

Comment:
This is unused, no?

```suggestion

```

Reply:
Oh yes, you are right. Relict from differenct implementation approaches....